api/v1alpha1/envoyproxy_types.go

```diff
@@ -93,6 +93,11 @@ type EnvoyProxySpec struct {
 	// +optional
 	Shutdown *ShutdownConfig `json:"shutdown,omitempty"`
 
+	// Init defines configuration for the envoy initialization process.
+	//
+	// +optional
+	Init *InitConfig `json:"init,omitempty"`
```

Comment:
can we start w/o this config ?

Reply:
From an earlier discussion it was suggested that there be an opt-in/out in EnvoyProxy, with the default being disabled upon initial release. Would you suggest a different way for handling that?

I was planning on using`ZoneOverride` for e2e tests but I imagine could handle that another way so wouldn't be opposed to dropping it.

I can't think of a need for changing `ConfigPath` so can remove that as well.

Reply:
I had assumed it would be something along the lines of `ZoneAwareRouting bool` , we could make it a struct so its easier to extend it